libr/socket/socket.c

```diff
@@ -225,14 +240,22 @@ R_API bool r_socket_connect (RSocket *s, const char *host, const char *port, int
 	sa.sin_family = AF_INET;
 	he = (struct hostent *)gethostbyname (host);
 	if (he == (struct hostent*)0) {
+#ifdef _MSC_VER
```

Comment:
No. Windows defined for 1) MinGW32 on Linux 2) Mingw32 on Windows 3) Cygwin on Windows.
_MSC_VER is very specific for MSVC. Probably aliasing it with "__MSVC__" can be better.